pkg/simple/client/monitoring/prometheus/promql.go

```diff
@@ -448,3 +465,37 @@ func makePVCMetricExpr(tmpl string, o monitoring.QueryOptions) string {
 	}
 	return strings.Replace(tmpl, "$1", pvcSelector, -1)
 }
+
+func makeIngressMetricExpr(tmpl string, o monitoring.QueryOptions) string {
+	var ingressSelector string
+	var jobSelector string
+	duration := "5m"
+
+	// parse Range Vector Selectors metric{key=value}[duration]
+	if o.MeterOptions != nil {
+		duration = o.MeterOptions.Step.String()
+	}
```

Comment:
It is not really good that the `step` parameter in range query specify both the step of the range query and time window of range vector

Reply:
@junotx Refined. Added new parameter "duration" to the query string.